app/models/environment.rb

```diff
@@ -96,6 +128,23 @@ def installation
     repository.installation
   end
 
+  def check_config!
+    fail("This repository is configured from .slashdeploy.yaml") if repository && repository.config?
```

Comment:
Yeah, this will throw an exception. It's mostly here, since calling these methods on a repo that's configured with a `.slashdeploy.yml` would be a programmer error. This should never actually get hit, but I wan't to fail loudly if it does.

Reply:
Cool.